htdocs/js/pages/entry/new.js

```diff
@@ -195,7 +188,7 @@ var postForm = (function($) {
             } else {
                 $custom_edit_button.hide();
             }
-        }).triggerHandler("change", rememberInitialValue);
+        });
```

Comment:
I think this introduces a bug in the JS. When testing, I noticed that if you are posting while logged out, and you:

1. select a non-public entry security, then:
2. type the name of the journal to post in,
3. the security selection will jump back to public as soon as you tab out of the journal field.

When I tried the same trick on the live site, the security dropdown stayed where it was.

I suspect this line is where I've gone wrong. Does anyone know why it depended on opts.spellcheck before? What should the correct value be here?